frontend/src/features/env/SwitchEnvFeedbackModal.tsx

```diff
@@ -33,6 +39,14 @@ export interface SwitchEnvModalProps {
   onChangeEnv: () => void
 }
 
+export const ADMIN_RADIO_OPTIONS = [
+  'I couldn’t find a feature I needed',
+  'The new FormSG did not function properly',
+]
+export const PUBLIC_RADIO_OPTIONS = ['I couldn’t submit my form']
+export const COMMON_RADIO_OPTIONS = ['I’m not used to the new FormSG']
+export const FEEDBACK_OTHERS_INPUT_NAME = 'others-input'
```

Comment:
nit: suggest something less likely to collide, maybe `react-feedback-others-input`?

Reply:
> lgtm with some changes (not requested, you can do if you want, since this is temporary)

ok tq! made them anyway since they're quick ones

Reply:
> nit: suggest something less likely to collide, maybe `react-feedback-others-input`?

https://github.com/opengovsg/FormSG/pull/5245/commits/215307aefcbc89beeab0ae70e456032cb73fc71f